src/test/java/internship/framework/pages/Homepage.java

```diff
@@ -21,6 +20,9 @@
     @FindBy(xpath = "//ul[contains(@class,'header-nav-global__list header-nav-global__engagement')]//li[3]//a[1]")
     private WebElement registerButton;
 
+    @FindBy(css = "a[title='My Account']")
+    private WebElement myAccountButton;
```

Comment:
this is duplicated in LoginPage:
 @FindBy(xpath = "//a[@title='My Account']")
    private WebElement myAccountLabel;
either move it there or move that one here.

Reply:
Fixed.